### PR TITLE
expression/aggregation: migrate test-infra to testify for util_test.go

### LINKED ISSUE #27909: migrate test-infra to testify for `expression/aggregation/util_test.go`
This is a subtask of #27904

### PR DESCRIPTION
**Pull request by penthaapatel**
### What problem does this PR solve?

Issue Number: close #27909

### What is changed and how it works?

What's Changed:

- migrate test-infra to testify for `expression/aggregation/util_test.go`
- add `main_test.go` file

### Release note 
```release-note
None
```

### CODE REVIEW

File: expression/aggregation/util_test.go [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
 )
 
-var _ = check.Suite(&testUtilSuite{})
-
-type testUtilSuite struct {
-}
-
-func (s *testUtilSuite) TestDistinct(c *check.C) {
+func TestDistinct(t *testing.T) {
 	sc := &stmtctx.StatementContext{TimeZone: time.Local}
```
Comment (tisonkun) [inline expression/aggregation/util_test.go:13]: I guess this case can be run in parallel. Please confirm.

```suggestion
func TestDistinct(t *testing.T) {
	t.Parallel()
	sc := &stmtctx.StatementContext{TimeZone: time.Local}
```

Comment (penthaapatel) [reply expression/aggregation/util_test.go]: Fixed it - ffaa1a9

**Review by tisonkun — APPROVED** [review]
Thanks for your contribution ! LGTM.

/cc @ichn-hu

**Review by ichn-hu — APPROVED** [review]
lgtm

**Comment by tisonkun** [comment]
@penthaapatel if you find failing tests instead of test time exceed 5 seconds, feel free to file an issue under https://github.com/pingcap/tidb/issues/25899 .

For the 5 seconds limitation, we're trying to stabilize the CI environment.

**Comment by tisonkun** [comment]
It seems failed with #27889 which is a known unstable test. Let's wait a bit for a fix...

**Comment by tisonkun** [comment]
> It seems failed with #27889 which is a known unstable test. Let's wait a bit for a fix...

I don't think this comment has magic 😄 . Actually it's better to be blocked so that we have more motivation to fix the issue. But for this PR, it's fine to get merge as the work itself is good.

**Pull request merged**